### ISSUE TITLE
Bad page state in process omicli

### ISSUE THREAD

**Issue by johanburati** [microsoft/OMS-Agent-for-Linux#1081]
I know there is a known [kernel bug](https://bugs.centos.org/view.php?id=15181) but since version **1.12.5** of the **OmsAgentForLinux** extension has been released, there are a lot of occurences of this bug.

Other processes are hitting the bug, but it seems it always start with **omicli**:
  
```
Oct 26 03:19:45 azurevm01 kernel: BUG: Bad page state in process omicli  pfn:4d77ff
Oct 26 03:20:07 azurevm01 kernel: BUG: Bad page state in process sh  pfn:3a84b5
Oct 26 03:20:07 azurevm01 kernel: BUG: Bad page state in process sh  pfn:3a84b5
Oct 26 03:20:08 azurevm01 kernel: BUG: Bad page state in process sh  pfn:3a84b5
Oct 26 03:20:08 azurevm01 kernel: BUG: Bad page state in process sh  pfn:3a84b5
```

I could see the issue on RHEL 7 with different kernels:

- 3.10.0-862.2.3.el7.x86_64
- 3.10.0-862.14.4.el7.x86_64
- 3.10.0-957.21.3.el7.x86_64
- 3.10.0-957.27.2.el7.x86_64
- 3.10.0-1062.1.2.el7.x86_64
- 3.10.0-1062.4.1.el7.x86_64

Will follow up internally.

**Comment by hestolz** [comment]
Johan please open the issue in [this repo](https://github.com/microsoft/omi/issues); omicli is a subcomponent of oms and this issue must be looked at by the appropriate team. Thanks.

**Comment by johanburati** [comment]
Ok, I have opened this issue: https://github.com/microsoft/omi/issues/650

**Comment by johanburati** [comment]
@hestolz  
That would be good if you could leave this ticket open for a while because this is a ongoing issue, a lot of people are affected by this and some of them will be looking for solution on github.
An open issue is easier to find than a closed one and it's easy to get lost between those three repos: OMS-Agent-for-Linux , azure-linux-extensions and omi

**Comment by hestolz** [comment]
We have applied the mitigating fix in #1095 , please monitor RH channel for kernel bug fix. We have not had any reports of this problem on newest version 1.12.15, so we will close this now.
